marklogic-data-hub/src/test/java/com/marklogic/hub/core/HubProjectTest.java

```diff
@@ -92,8 +92,8 @@ public void testInit() throws IOException {
         assertTrue(new File(projectPath, "src/main/hub-internal-config/security/roles/data-hub-step-definition-writer.json").exists());
         assertTrue(new File(projectPath, "src/main/hub-internal-config/security/roles/data-hub-entity-model-reader.json").exists());
         assertTrue(new File(projectPath, "src/main/hub-internal-config/security/roles/data-hub-entity-model-writer.json").exists());
-        assertTrue(new File(projectPath, "src/main/hub-internal-config/security/roles/data-hub-saved-query-reader.json").exists());
-        assertTrue(new File(projectPath, "src/main/hub-internal-config/security/roles/data-hub-saved-query-writer.json").exists());
+        assertTrue(new File(projectPath, "src/main/hub-internal-config/security/roles/hub-central-saved-query-reader.json").exists());
```

Comment:
It's fine to do this, but I think soon, we should remove all of these assertions. If the roles/privileges weren't correctly written out, that "bootstrap" would fail and none of the tests would run. So these assertions are redundant and not adding any value. Fine to keep them for now, I'll look into removing all of them soon.